provider/aws/dist/rack.json

```diff
@@ -1337,7 +1337,7 @@
             "bootcmd:\n",
             "  - mkswap /dev/xvdb\n",
             "  - swapon /dev/xvdb\n",
-            "  - yum install -y aws-cli nfs-utils\n",
+            "  - until yum install -y aws-cli nfs-utils; do echo \"Waiting for network\"; done;\n",
```

Comment:
We're relying on CF for the overall timeout and for the loops yum has it's own 10 second timout (#1633 cc @mattmanning ).